index.html

```diff
@@ -0,0 +1,196 @@
+            </span>
+          </li>
+          <li>
+            <span class="date">1938</span> -
+            <span class="detail"
+              >Marries wife of 69 years Margret Gibson. Gets laid off due to
+              budget cuts. Inspired by Elvin Charles Stakman, he returns to
+              school study under Stakman, who teaches him about breeding
+              pest-resistent plants.
+            </span>
+          </li>
+          <li>
+            <span class="date">1941</span> -
+            <span class="detail"
+              >Tries to enroll in the military after the Pearl Harbor attack,
+              but is rejected. Instead, the military asked his lab to work on
+              waterproof glue, DDT to control malaria, disinfectants, and other
+              applied science.
+            </span>
+          </li>
+          <li><span class="date">1942</span> - <span class="detail"></span></li>
+          <li>
+            <span class="date"></span
+            ><span class="detail"
+              >Receives a Ph.D. in Genetics and Plant Pathology
+            </span>
+          </li>
+          <li>
+            <span class="date">1944</span> -
+            <span class="detail"
+              >Rejects a 100% salary increase from Dupont, leaves behind his
+              pregnant wife, and flies to Mexico to head a new plant pathology
+              program. Over the next 16 years, his team breeds 6,000 different
+              strains of disease resistent wheat - including different varieties
+              for each major climate on Earth.
+            </span>
+          </li>
+          <li>
+            <span class="date">1945</span> -
+            <span class="detail"
+              >Discovers a way to grown wheat twice each season, doubling wheat
+              yields
+            </span>
+          </li>
+
+            <span class="date">1953</span>
+            <span class="date">1953</span>
+            <span class="detail"
+              >crosses a short, sturdy dwarf breed ofwheat with a high-yeidling
+              American breed, creating a strain that responds well to
+              fertilizer. It goes on to provide 95% of Mexico's wheat</span
+            >
+          </li>
+          <li>
+            <span class="date">1962</span> -
+            <span class="detail"
+              >Visits Delhi andrings his high-yielding strains of wheat to the
+              Indian subcontinent in time to help mitigate mass starvation due
+              to a rapidly expanding population
+            </span>
+          </li>
+          <li>
+            <span class="date">1970</span> -
+            <span class="detail">receives the Nobel Peace Prize</span>
+          </li>
+          <li>
+            <span class="date">1983</span> -
+            <span class="detail"
+              >helps seven African countries dramatically increase their maize
+              and sorghum yields
```

Comment:
Add a tag as `h4`

Reply:
Kindly add a p tag in place of span tags. As mentioned previously p tags enable differentiation of the text levels optimizing SEO :heavy_check_mark: